solr/core/src/java/org/apache/solr/jersey/RequestMetricHandling.java

```diff
@@ -40,9 +39,18 @@
 /**
  * A request and response filter used to initialize and report per-request metrics.
  *
- * <p>Currently, JAX-RS v2 APIs rely on a {@link
- * org.apache.solr.handler.RequestHandlerBase.HandlerMetrics} instance from an associated request
- * handler.
+ * <p>Currently, Jersey resources that have a corresponding v1 API produce the same metrics as their
+ * v1 equivalent and rely on the v1 requestHandler instance to do so. Solr facilitates this by
+ * building a map of the JAX-RS resources to requestHandler mapping (a {@link
+ * PluginBag.JaxrsResourceToHandlerMappings}), and using that to look up the associated request
+ * handler (if one exists) in pre- and post- filters
```

Comment:
```suggestion
 * org.apache.solr.core.PluginBag.JaxrsResourceToHandlerMappings}), and using that to look up the
 * associated request handler (if one exists) in pre- and post- filters
```

apparently just need to be fully qualified.

Reply:
The suggestion includes the `tidy` reformatting too.

Reply:
Thanks Kevin! LGTM.

(This "Suggested change" feature in Github is pretty cool; I always forget it's there.)